copter/source/docs/weathervaning.rst

```diff
@@ -0,0 +1,73 @@
+parameter to a non-zero value. The default is not to use active
+weathervaning.
+
++---------------------------------------------+--------------------------+
++ :ref:`WVANE_ENABLE<WVANE_ENABLE>` Value     | Weathervane Direction    +
++=============================================+==========================+
++         0                                   |  Disabled                +
++---------------------------------------------+--------------------------+
++         1                                   |  Nose Into Wind(default) +
++---------------------------------------------+--------------------------+
++         2                                   |  Nose OR Tail Into Wind  +
++---------------------------------------------+--------------------------+
++         3                                   |  Side Into Wind          +
++---------------------------------------------+--------------------------+
++         4                                   |  Tail Into Wind          +
++---------------------------------------------+--------------------------+
+
+Then you must enable it for GUIDED and/or AUTO mode by setting bit 7 of the :ref:`GUID_OPTIONS<GUID_OPTIONS>` and/or :ref:`AUTO_OPTIONS<AUTO_OPTIONS>` parameter, respectively. It will not be active in any other 
+mode.
+
+.. note:: enabling weathervaning will override any yaw specific control in missions, such as pointing the nose to ROI targets, or responding to MAVLink YAW positioning commands. If pilot yaw control is enabled, the pilot can change the yaw, but as soon as pilot input ceases, the vehicle will yaw back to its Weathervaing Direction set above.
+
+The way it works is the autopilot looks at the roll and/or pitch attitude needed to
+control the desired position. The basic algorithm is "yaw into the
+roll/pitch". If the aircraft needs hold roll to the right in order to hold
+position then it will turn in that direction on the assumption that
+the right roll is needed in order to hold against the wind (assuming nose into the wind). Similarly, for Side Into the Wind, it will use the pitch angle and yaw appropriately, trying to zero the pitch required to hold position.
+
+.. note:: by default, weathervaning does not use pitch, only roll. This is to prevent unwanted continuous yawing if the hover attitude has not been trimmed to hover, in place, with no wind. If this **has** been done, then pitch driven weathervaning can be enabled by setting :ref:`WVANE_OPTIONS<WVANE_OPTIONS>` bit 0 to "1". This will speed weathervaning if positioned with the wind blowing from behind (ie mostly being held in position with pitch, not roll). This does not affect Side Into Wind, it always uses pitch, hover trim attitude must be properly set for good operation.
+
+How quickly the aircraft yaws is determined by the :ref:`WVANE_GAIN<WVANE_GAIN>`
+parameter. It converts the lean angle into degs/sec of yaw. A good value to start with is 1 (1 degree roll = 1 deg/sec yaw). Higher values will make
```

Comment:
"1 degree roll = 1 deg/sec yaw" I have not checked, but that is probably not how the math works out.

Reply:
that's historical, if incorrect, we need to change here and in QPlane...can you do that?

Reply:
its valid for copter...need to update QPlane wiki..